dev/com.ibm.ws.kernel.feature_fat/publish/servers/com.ibm.ws.kernel.feature.fat.versionless.minify/server.xml

```diff
@@ -0,0 +1,10 @@
+<server>
+
+    <include location="../fatTestPorts.xml"/>
+    
+    <featureManager>
+		<platform>jakartaee-10.0</platform>
+        <feature>servlet</feature>
```

Comment:
I don't like the formatting, please fix the spacing. It isn't actually testing the scenario that found this problem  You need a feature that has more than one platform version and you pick a later platform that isn't the first one in the list of platforms for that feature.  So if you added ejb in the feature list that would generate the original problem.

Reply:
agreed - will add ejb...